rir/src/compiler/pir/instruction.cpp

```diff
@@ -510,6 +510,14 @@ Instruction* BuiltinCallFactory::New(Value* callerEnv, SEXP builtin,
         return new CallBuiltin(callerEnv, builtin, args, srcIdx);
 }
 
+VisibilityFlag CallBuiltin::visibilityFlag() const {
+    return (VisibilityFlag)getFlag(builtinId);
```

Comment:
that is way too clever.

the dumber the better to read.

```
if (flag == 1)
  return VisibilityFlag::Off;
if (flag == 0)
  return VisibiliyFlag::On;
return VisibilityFlag::Unknown;
```

is so much easier to read an 100x more robust